packages/components/src/mobile/bottom-sheet/link-suggestion-item-cell.native.js

```diff
@@ -58,6 +61,7 @@ function LinkSuggestionItemCell( { suggestion, onLinkPicked } ) {
 
 	return (
 		<Cell
+			{ ...props }
```

Comment:
I am not a big fan of this pattern {...props} mostly since it makes it harder to find the props that are being passed down. 
Do we need to pass down all the props? Why are we doing this in this case?

Reply:
I realize we probably do this in other parts of the code base and this might just be my preference :)

Reply:
It's not my personal preference either. I just also don't feel particularly strongly about either direction.

Reply:
I will resolve this in a smaller future PR. I appreciate your help releasing the current changes, @enejb .